pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -1284,52 +1284,74 @@ func TestImportMysql(t *testing.T) {
 	multitable := getMultiTableMysqlDumpTestdata(t)
 	multitable = fmt.Sprintf("nodelocal://%s", strings.TrimPrefix(multitable, baseDir))
 
+	const expectAll, expectSimple, expectSecond = "all", "simple", "multitable"
+
 	for _, testCase := range [][]string{
-		{`read data only`, simple, `IMPORT TABLE simple (i INT PRIMARY KEY, s text, b bytea) MYSQLDUMP DATA ($1)`},
```

Comment:
i'd love to see a test with two input files. we've had errors with those before so we should make sure they don't occur again.

Reply:
the syntax only accepts one file right now.